src/Controller/SecurityController.php

```diff
@@ -69,6 +74,13 @@ public function register(Request $request, UserPasswordEncoderInterface $passwor
             $entityManager->persist($user);
             $entityManager->flush();
 
+            $email = (new Email())
+            ->from(strval($this->getParameter('mailer_from')))
+            ->to('your_email@example.com')
```

Comment:
wouldn't it be better to send your mail to the email address of the person that just registered instead of always this placeholder address?